brooklyn-server/software/base/src/main/java/org/apache/brooklyn/entity/software/base/lifecycle/MachineLifecycleEffectorTasks.java

```diff
@@ -140,7 +141,10 @@ public void attachLifecycleEffectors(Entity entity) {
      * the behaviour in this lifecycle class instance.
      */
     public Effector<Void> newStartEffector() {
-        return Effectors.effector(Startable.START).impl(newStartEffectorTask()).build();
+        return Effectors.effector(Startable.START)
+                .parameter(StartSoftwareParameters.SKIP_INSTALLATION)
```

Comment:
A better place for this would be the `SoftwareProcessDriverLifecycleEffectorTasks` (overriding the method there), but given that the rest of the lifecycle effectors set their process related parameters here ok to stay here for consistency.